vignettes/Model.Rmd

```diff
@@ -14,86 +14,282 @@ knitr::opts_chunk$set(
+-   `natural_deaths`: deaths from old age
+-   `S_count`, `A_count`, `D_count`, `U_count`, `Tr_count`: the human population size in each state
+-   `ica_mean`: mean acquired immunity to clinical infection
+-   `icm_mean`: mean maternal immunity to clinical infection
+-   `ib_mean`: mean blood immunity to all infection
+-   `id_mean`: mean immunity from detected using microscopy
+-   `iva_mean`: mean acquired immunity to severe infection
+-   `ivm_mean`: mean maternal immunity to severe infection
+-   `n_730_3650`: population size of an age group of interest (where the default is set to 730-3650 days old, or 2-10 years, but which may be adjusted (see [Demography](https://mrc-ide.github.io/malariasimulation/articles/Demography.html) vignette for more details)
+-   `n_detect_730_3650`: number with possible detection through microscopy of a given age group
+-   `p_detect_730_3650`: the sum of probabilities of detection through microscopy of a given age group
+-   `E_All_count`, `L_All_count`, `P_All_count`, `Sm_All_count`, `Pm_All_count`, `Im_All_count`, `NonExistent_All_count`: mosquito population sizes in each state
+-   `total_M_All`: number of adult mosquitoes
 
-The human variables are documented in "R/variables.R".
+```{r, output.lines=6}
+head(test_sim, n = 3)
+```
 
-The human biological processes are spread out between the following files:
+Additional output details can be found in the `run_simulation()` documentation.
 
- 1. 'human_infection.R' (orange)
- 2. 'disease_progression.R' (green)
- 3. 'mortality_processes.R'(red)
+### Additional outputs
 
-```{r echo=FALSE}
-DiagrammeR::grViz("digraph {
-  graph [layout = dot, rankdir = LR]
+**Age stratified** results for **incidence**, **clinical incidence** and **severe case incidence** may also be included in the output if desired and must be specified in the parameter list (see `get_parameters()` for more details and [Demography](https://mrc-ide.github.io/malariasimulation/articles/Demography.html) for an example). These inputs will add extra columns to the output for the number of infections (**`n_`**) and the sum of probabilities of infection (**`p_`**) for the relevant total, clinical or severe incidences for each specified age group.
+
+Where **treatments** are specified, `n_treated` will report the number that have received treatment. Where **bed nets** are distributed, `net_usage` specifies the number sleeping under a bednet.
+
+### Output visualisation
+
+These outputs can then be visualised, such as the population changes in states. Another key output is the prevalence of detectable infections between the ages of 2-10 (*Pf*PR~2-10~), which can be obtained by dividing `n_detect_730_3650` by `n_730_3650`.
+
+```{r, fig.align = 'center', fig.height = 5, fig.width = 7}
+# Set color palette
+cols  <- c("#E69F00", "#56B4E9", "#009E73", "#F0E442", "#0072B2", "#D55E00", "#CC79A7")
+
+# Define vector of column names to plot
+cols_to_plot <- paste0(c("S","D","A","U","Tr"),"_count")
```

Comment:
The U and Tr legend lines aren't showing up when I knit, but not a game changer

Reply:
I've put these on separate lines now.